pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -1425,7 +1425,7 @@ func (gce *GCECloud) AddSSHKeyToAllInstances(user string, keyData []byte) error
 					glog.Info("SSHKey already in project metadata")
 					return true, nil
 				}
-				value := *item.Value + "\n" + keyString
+				value := keyString + "\n" + *item.Value
```

Comment:
Not actually necessary. I can revert it.

My original thought was that it would keep our key closer to the front of the project metadata to protect against some other writer truncating our key off, leaving us without enough room to write it back. There isn't any evidence that is ever happening.